### DISCUSSION TITLE
How can I deploy a .NET Aspire application in a local network?

### DISCUSSION THREAD

**Discussion by kurzenkov** [microsoft/aspire#6993] [Q&A] [answered]
Hello everyone! I'm wondering if there's a way to deploy a .NET Aspire application directly from my development environment into a Kubernetes cluster that isn't running on my local machine but is still part of the local network. If so, could you please share an example or a link? Thanks!

**Comment by davidfowl** [comment]
https://learn.microsoft.com/en-us/dotnet/aspire/deployment/overview#the-aspir8-project

**Reply by davidfowl** [reply, marked as answer]
With 9.2 there's improvements to deployment https://learn.microsoft.com/en-us/dotnet/aspire/whats-new/dotnet-aspire-9.2#-deployment-improvements
